src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__cells calendar__cells--last-months-day--31">
+        <div class="calendar__cell calendar__cell--start-day--mon"></div>
+        <div class="calendar__cell calendar__cell--start-day--tue"></div>
+        <div class="calendar__cell calendar__cell--start-day--wed"></div>
+        <div class="calendar__cell calendar__cell--start-day--thu"></div>
+        <div class="calendar__cell calendar__cell--start-day--fri"></div>
+        <div class="calendar__cell calendar__cell--start-day--sat"></div>
+        <div class="calendar__cell calendar__cell--start-day--sun"></div>
```

Comment:
Add the modifier for start day on the 11 line
```suggestion
        <div class="calendar__cell"></div>
        <div class="calendar__cell"></div>
        <div class="calendar__cell"></div>
        <div class="calendar__cell"></div>
        <div class="calendar__cell"></div>
        <div class="calendar__cell"></div>
        <div class="calendar__cell"></div>
```